tests/20-SideMenu/BoardsMenuSection.test.tsx

```diff
@@ -0,0 +1,91 @@
+import "@testing-library/jest-dom/extend-expect";
+
+import * as stories from "stories/20-SideMenu/00-BoardsMenuSection.stories";
+
+import {
+  Screen,
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+  within,
+} from "@testing-library/react";
+
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { composeStories } from "@storybook/testing-react";
+import { mocked } from "ts-jest/utils";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("@storybook/addon-actions");
+
+const { Regular, Empty, Loading, Long } = composeStories(stories);
+
+describe("Regular", () => {
+  test("Renders section with board menu items", async () => {
+    render(<Regular />);
+  
+    //TODO: fill this
+  });
+  
+  //Not sure if this should be it's own test or part of the previous test
```

Comment:
I don't know actually, I assume people think differently on this. Keeping it separate seems safe.